docs/available_commands.md

```diff
@@ -300,7 +296,7 @@ When passing the targets, names and values in a file, the file must conform to t
             "id": {
                 # Could be any one of the identifying properties in CycloneDX.
                 # Multiple identifiers are not allowed (with the special exception of name,
-                # group and version/version_range which are only valid together)
+                # group and version which are only valid together)
```

Comment:
Have you intentionally reverted this change?

Reply:
No, another remnant from the merge conflict.